File: debot/src/main/java/com/tomoima/debot/DebotStrategies.java
package com.tomoima.debot;

import com.tomoima.debot.strategy.DebotStrategy;

import java.util.ArrayList;

public class DebotStrategies {
    private static ArrayList<DebotStrategy> strategies;

    public void initialize(ArrayList<DebotStrategy> strategies){
        this.strategies = strategies;
    }

    public ArrayList<DebotStrategy> getStrategies(){
        return strategies;
    }
}

File: debot/src/main/java/com/tomoima/debot/DebotStrategies.kt
package com.tomoima.debot

import com.tomoima.debot.strategy.DebotStrategy
import java.util.*

class DebotStrategies {

    companion object {
        private var strategies = ArrayList<DebotStrategy>()

        fun initialize(strategies: ArrayList<DebotStrategy>) {
            this.strategies = strategies
        }

        fun get(): ArrayList<DebotStrategy> {
            return strategies
        }
    }
}
